src/Umbraco.Web/PropertyEditors/MediaCropperPropertyEditor.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using Umbraco.Core;
+using Umbraco.Core.PropertyEditors;
+
+namespace Umbraco.Web.PropertyEditors
+{
+    /// <summary>
+    /// Media picker property editors that stores crop data
+    /// </summary>
+    [PropertyEditor(Constants.PropertyEditors.MediaCropperAlias, "Media Cropper", PropertyEditorValueTypes.Text, "mediacropper", IsParameterEditor = true, Group = "media", Icon = "icon-crop")]
+    public class MediaCropperPropertyEditor : PropertyEditor
+    {
+        public MediaCropperPropertyEditor()
+        {
+            InternalPreValues = new Dictionary<string, object>
+            {
+                {"idType", "udi"},
```

Comment:
Do we need to track the `idType`?

Reply:
Probably not, this is just mimicking the original image cropper and media picker, which probably has this for backwards compatibility.